code/game/machinery/vending.dm

```diff
@@ -1142,7 +1141,6 @@
 	desc = "Wall-mounted Medical Equipment dispenser."
 	icon_state = "wallmed"
 	icon_deny = "wallmed-deny"
-	req_access = list(access_medical)

```

Comment:
Forgive me if I'm mistaken, but isn't this for the wallmed vendors found inside medical itself?

Reply:
Pretty sure the wallmed vendors have near identical items to the emergency nanomeds so I don't see how it can hurt to allow non-medical access to these

Reply:
Would you like that change rolled back then to keep the EmergencyNanomeds in medical doctor access only?

Reply:
I personally see no reason for certain Emergency Nanomeds to be medical only. It should all be public access. Frankly it makes even more sense the ones IN medbay are public access for players to use on themselves and others in times of emergency.